src/h3lib/lib/h3Index.c

```diff
@@ -266,6 +266,9 @@ H3Index H3_EXPORT(h3ToCenterChild)(H3Index h, int childRes) {
  */
```

Comment:
Can we be explicit about what codes are returned in success and failure cases?

Reply:
So far we haven't done that in the public API, I think with the idea that the error codes might change in patch changes. The error codes have been used in some of the bindings to generate exception messages, though, so they are useful.